### ISSUE TITLE
Install apple intelligent permanently ( UNSUPPORTED DEVICE )

### ISSUE THREAD

**Issue by Ak-2077** [leminlimez/Nugget#224]
Step 1. Spoof to 16promax.
 Step 2. Wait for it to download AI model ,open Apple Intelligence settings screen for it to download faster
 Step 3 .Test and confirm AI is running.
Step 4 . (Very important ) before you spoof to "original" , here is few step 
     1 . DO NOT  turn Off AI from setting 
     2 . Turn On Airplane mode (before spoofing to none )
     3. Remove All App running in background 
     4. open nugget and go to eligibility page and turn on ai and spoof to none 
 Step 5 . As soon as iPhone reboots,Stay in airplane mode and test Siri after 2 **minutes.**
 Step 6 . Once new Siri confirmed working a few times, stay on Airplane mode for 3-5 more minutes 
Step 7 . Most important :- test your Siri every 10-20 seconds (while airplane mode is on )
 Step . 8 Turn off Airplane mode AFTER 10 MIN 
 Step 9 . VERY VERY IMPORTANT :- **NEVER TOUCH Apple Intelligence Settings ever again**.
  
Enjoy:)

**Comment by Ak-2077** [comment]
NO it won't go after power off 
Follow All step I have given because I already using since 2 week

**Comment by MasterMind-del** [comment]
> NO it won't go after power off Follow All step I have given because I already using since 2 week

Turn off your and testify what I’m saying

**Comment by MasterMind-del** [comment]
> NO it won't go after power off Follow All step I have given because I already using since 2 week

Everybody has its method. #185 is my method

**Comment by Ak-2077** [comment]
yes I agree that was your method but there was a catch THAT I HAVE RESOLVED

**Comment by MasterMind-del** [comment]
> yes I agree that was your method but there was a catch THAT I HAVE RESOLVED

Did you off your phone?

**Comment by MasterMind-del** [comment]
> yes I agree that was your method but there was a catch THAT I HAVE RESOLVED
Power off your phone and tell us if is still working or not. And if it working then I will love to close my solution so that people can start using yours as new method

**Comment by MasterMind-del** [comment]
> > yes I agree that was your method but there was a catch THAT I HAVE RESOLVED


> > Power off your phone and tell us if is still working or not. And if it working then I will love to close my solution so that people can start using yours as new method

**Comment by Ak-2077** [comment]
I did twice

**Comment by chuck00000** [comment]
When you restart or power off your iPhone again manually afterwards , even though you don't open the Apple Intelligence Settings, airplane mode is still on before restart, The Siri will be revert old siri

**Comment by MasterMind-del** [comment]
> When you restart or power off your iPhone again manually afterwards , even though you don't open the Apple Intelligence Settings, airplane mode is still on before restart, The Siri will be revert old siri

Yes, that’s what I’m trying to tell him but I don’t think he’s getting my point here

**Comment by Agustine808** [comment]
> Step 1. Spoof to 16promax. Step 2. Wait for it to download AI model ,open Apple Intelligence settings screen for it to download faster Step 3 .Test and confirm AI is running. Step 4 . (Very important ) before you spoof to "original" , here is few step 1 . DO NOT turn Off AI from setting 2 . Turn On Airplane mode (before spoofing to none ) 3. Remove All App running in background 4. open nugget and go to eligibility page and turn on ai and spoof to none Step 5 . As soon as iPhone reboots,Stay in airplane mode and test Siri after 2 **minutes.** Step 6 . Once new Siri confirmed working a few times, stay on Airplane mode for 3-5 more minutes Step 7 . Most important :- test your Siri every 10-20 seconds (while airplane mode is on ) Step . 8 Turn off Airplane mode AFTER 10 MIN Step 9 . VERY VERY IMPORTANT :- **NEVER TOUCH Apple Intelligence Settings ever again**.
> 
> Enjoy:)

if you reboot the phone anyhow apple intelligence will be gone!

**Comment by Agustine808** [comment]
> > When you restart or power off your iPhone again manually afterwards , even though you don't open the Apple Intelligence Settings, airplane mode is still on before restart, The Siri will be revert old siri
> 
> Yes, that’s what I’m trying to tell him but I don’t think he’s getting my point here

yup thats what exactly happening!

**Comment by Mahrad2009YP** [comment]
hi
I unfortunately join to apple intelligence waitlist when my device isn't spoofed.
what will happens for my device?
please help me

**Comment by chuck00000** [comment]
Notice: don't touch " Writing tools" option, it will revert Old Siri immediately!

**Comment by MasterMind-del** [comment]
> Notice: don't touch " Writing tools" option, it will revert Old Siri immediately!

As for that’s one I don’t believe, cause I’m using writing tools every day

**Comment by chuck00000** [comment]
> > Notice: don't touch " Writing tools" option, it will revert Old Siri immediately!
> 
> As for that’s one I don’t believe, cause I’m using writing tools every day

But today I touched "Writing tools", after that  immediately revert old Siri.


But "writing tools" on unsupported Devices is just UI, the function is not working

**Comment by MasterMind-del** [comment]
> > > Notice: don't touch " Writing tools" option, it will revert Old Siri immediately!
> > 
> > 
> > As for that’s one I don’t believe, cause I’m using writing tools every day
> 
> But today I touched "Writing tools", after that immediately revert old Siri.
> 
> But "writing tools" on unsupported Devices is just UI, the function is not working

Yes, but you can use composer in writing tools and it’s will work fine. Apart from that you can’t use any other features in writing tools

**Comment by chuck00000** [comment]
Apple Intelligence often revert old Siri after at least 3 ~5 days, even though its not touched Apple Intelligence Settings menu,the phone haven't restart yet. I have sproof again.

**Comment by chuck00000** [comment]
> > > > Notice: don't touch " Writing tools" option, it will revert Old Siri immediately!
> > > 
> > > 
> > > As for that’s one I don’t believe, cause I’m using writing tools every day
> > 
> > 
> > But today I touched "Writing tools", after that immediately revert old Siri.
> > But "writing tools" on unsupported Devices is just UI, the function is not working
> 
> Yes, but you can use composer in writing tools and it’s will work fine. Apart from that you can’t use any other features in writing tools

Apple Intelligence often revert old Siri after at least 3 ~5 days, even though its not touched Apple Intelligence Settings menu,the phone haven't restart yet. I have sproof again. Happened fourth times.

**Comment by MasterMind-del** [comment]
Nope, I used old versions of nuggets software 4.1 to spoofed my device to 16 pro max and it working more than two months, right now I can restart my device and still the new Siri is working fine.
